docs/get_started_with_qiskit.ipynb

```diff
@@ -0,0 +1,304 @@
+      ],
+      "text/plain": [
+       "<Figure size 454.517x284.278 with 1 Axes>"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit import QuantumCircuit\n",
+    "\n",
+    "# Create a new circuit with two qubits (first argument) and two classical\n",
+    "# bits (second argument)\n",
+    "qc = QuantumCircuit(2, 2)\n",
+    "\n",
+    "# Add a Hadamard gate to qubit 0\n",
+    "qc.h(0)\n",
+    "\n",
+    "# Perform a controlled-X gate on qubit 1, controlled by qubit 0\n",
+    "qc.cx(0, 1)\n",
+    "\n",
+    "# Measure qubit 0 to cbit 0, and qubit 1 to cbit 1\n",
+    "qc.measure(0, 0)\n",
+    "qc.measure(1, 1)\n",
+    "\n",
+    "# Return a drawing of the circuit using MatPlotLib (\"mpl\"). This is the\n",
+    "# last line of the cell, so the drawing appears in the cell output.\n",
+    "# Remove the \"mpl\" argument to get a text drawing.\n",
+    "qc.draw(\"mpl\")"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "0c957de9",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "See :class:`~.QuantumCircuit` in the documentation for all available operations."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "89c383e9-4d7b-4ba4-bc18-8960b9fcc923",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "## Run your quantum program\n",
+    "\n",
+    "Quantum circuits can produce random results, so you'll often want to collect a sample of the outputs by running the circuit many times. You can sample a circuit's output using the `Sampler` class. "
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "89f42d59-305c-4d6d-b6d4-c300afc91de6",
+   "metadata": {
+    "editable": true,
+    "raw_mimetype": "text/restructuredtext",
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "This tutorial uses Qiskit's built-in :class:`~.primitives.Sampler`, which simulates a perfect quantum computer, but can only handle small numbers of qubits. You can replace this with samplers of higher-performance simulators from `Qiskit Aer <https://github.com/Qiskit/qiskit-aer#readme>`__, or with samplers that send your circuit over the internet to run on real quantum computers (such as `IBM's Qiskit Runtime <https://docs.quantum-computing.ibm.com/api/qiskit-ibm-runtime/runtime_service>`__ sampler)."
```

Comment:
Would it be better to link to the aer doc's sampler page here? https://qiskit.org/ecosystem/aer/stubs/qiskit_aer.primitives.Sampler.html#qiskit_aer.primitives.Sampler